### ISSUE TITLE
Decreasing Points Total

### ISSUE THREAD

**Issue by AdventureBear** [FreeCodeCamp/freecodecamp#300]
My challenges total reset from 80 to 54! When I clicked on take me to my next challenge (all of which I've completed) it showed me the git challenge. I figured it was a glitch so hit "completed" and my number reset/recalculated and it took me to node, which I've also completed. It looks like the total is now only counting the challenges, but not the adding points from teh bonfires. It does correctly take me to my next bonfire. I just wanted to pass that along in case the total count is a bug, or if it's doing what it's supposed to do. I liked seeing the nubmer go up with each bonfire I completed.

**Comment by ItsThatHighlander** [comment]
I also experienced this issue. Upon completing the last bonfire, my score dropped from 50 or so to 16.

**Comment by TimRizzo** [comment]
Probably related- after the slack switch, the challenges menu was sending me back to that challenge. So I signed up for slack, marked the challenge complete and it appears that bonfires points were reset to 0.

**Comment by QuincyLarson** [comment]
This has been fixed on our ux-improvements branch that we're working to merge in. Your points will be automatically restored once this is merged in. Thanks for your patience.

**Comment by QuincyLarson** [comment]
I'm going to move this to done because it's fixed on the branch that will soon be merged into master and pushed to production.
